drivers/xen/9pfront/Config.uk

```diff
@@ -2,6 +2,7 @@ menuconfig LIB9PFRONT
 	bool "9pfront: Xen 9pfs volumes"
 	select LIBXENBUS
 	select LIBXENHEADERS
+	select LIBXENGNTTAB
```

Comment:
In #1517 `XEN_GNNTAB` was selected conditional to `XEN_PV || ARCH_ARM_64`. Should we be preserving that condition in drivers?